lib/cosmos/interfaces/interface.rb

```diff
@@ -53,6 +53,10 @@ class Interface
     #   read from the interface
     attr_accessor :routers
```

Comment:
I know you don't want to break APIs but any thought about changing this to tlm_routers?